packages/@okta/vuepress-theme-prose/layouts/AuthorizationLayout.vue

```diff
@@ -19,9 +19,13 @@
 </template>
 
 <script>
+
 export default {
   components: {
     Footer: () => import('../components/Footer.redesign')
+  },
+  mounted: async function() {
+        await import('../util/pendo');
```

Comment:
No reason to await here, nothing has to run afterwards

```suggestion
  },
  mounted: function() {
        import('../util/pendo');
```